src/test/java/seedu/momentum/logic/parser/AddCommandParserTest.java

```diff
@@ -64,31 +68,33 @@ public void parse_compulsoryFieldMissing_failure() {
         String expectedMessage = String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddCommand.MESSAGE_USAGE);
 
         // missing name prefix
-        assertParseFailure(parser, VALID_NAME_BOB,
-                expectedMessage);
+        assertParseFailure(parser, VALID_NAME_BOB + DESCRIPTION_DESC_BOB, expectedMessage);
 
-        // missing phone prefix
-        //        assertParseFailure(parser, NAME_DESC_BOB + VALID_PHONE_BOB,
-        //                expectedMessage);
+        // missing description prefix
+        assertParseFailure(parser, NAME_DESC_BOB + VALID_DESCRIPTION_BOB, expectedMessage);
 
         // all prefixes missing
-        assertParseFailure(parser, VALID_NAME_BOB,
-                expectedMessage);
+        assertParseFailure(parser, VALID_NAME_BOB + VALID_DESCRIPTION_BOB, expectedMessage);
     }
 
     @Test
     public void parse_invalidValue_failure() {
         // invalid name
-        assertParseFailure(parser, INVALID_NAME_DESC
+        assertParseFailure(parser, INVALID_NAME_DESC + DESCRIPTION_DESC_BOB
                 + TAG_DESC_HUSBAND + TAG_DESC_FRIEND, Name.MESSAGE_CONSTRAINTS);
 
+        // invalid description / empty description
+        assertParseFailure(parser, NAME_DESC_BOB + INVALID_DESCRIPTION_DESC
+                + TAG_DESC_HUSBAND + VALID_TAG_FRIEND,
+                        String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddCommand.MESSAGE_USAGE));
+
```

Comment:
Just to confirm, the only time this test will fail is when the description is empty right

Reply:
yup